go.mod

```diff
@@ -52,6 +52,7 @@ require (
 
 require (
 	cloud.google.com/go/storage v1.28.1
+	github.com/VividCortex/ewma v1.2.0
```

Comment:
We already have an ewma implementation at `pkg/util/math/rate.go`. Does the one we import give us a significant benefit over the one we already have?

Reply:
@pracucci I missed the existing one :/ Thanks for making me aware.